### PR TITLE
add atoum task

### PR DESCRIPTION
**Pull request by agallou**
This task allows to run [atoum](http://atoum.org/) tests inside robo.

```php
$this->taskAtoum()
	->debug()
	//->tags("databaseTest")
	//->lightReport()
	//->tap()
	//->files(["path/to/test1.php", "path/to/test2.php"])
	->directories('tests/units')
	//->configFile("config/dev.php")
	//->debug()
	->run();
```

### CODE REVIEW

File: src/Task/Testing/Atoum.php [hunk 1]
```diff
@@ -0,0 +1,170 @@
+<?php
+namespace Robo\Task\Testing;
+
+use Robo\Contract\CommandInterface;
+use Robo\Contract\PrintedInterface;
+use Robo\Task\BaseTask;
+
+/**
+ * Runs atoum tests
```
Comment (DavertMik) [inline src/Task/Testing/Atoum.php]: Please put all documentation here, the same way it is done for other tasks like PHPUnit.
Real docs will be generated from it

**Comment by DavertMik** [comment]
Thanks, nice addition. Could you update docblock for the task with documentation? It is necessary because all *.md docs are generated from docblocks

**Comment by agallou** [comment]
@DavertMik docblocks updated and doc generated using the "robo docs" task

**Comment by DavertMik** [comment]
Thank you!

**Pull request merged**